File: coupon-common/src/main/java/com/kakao/mobility/kmcoupon/exception/BadRequestException.java
package com.kakao.mobility.kmcoupon.exception;

public class BadRequestException extends RuntimeException {

    public BadRequestException() {
        super("잘못된 요청입니다.");
    }

    public BadRequestException(String message) {
        super(message);
    }

    public BadRequestException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: coupon-common/src/main/java/com/kakao/mobility/kmcoupon/exception/BadRequestException.kt
package com.kakao.mobility.kmcoupon.exception

open class BadRequestException : RuntimeException {
    constructor() : super("잘못된 요청입니다.") {}
    constructor(message: String) : super(message) {}
    constructor(message: String, cause: Throwable) : super(message, cause) {}
}